src/components/capture/create/index.tsx

```diff
@@ -356,52 +328,75 @@ function CaptureCreation() {
                     showValidation: true,
                 });
             } else {
-                // TODO (supabase) - `discovers:id=eq.${response.data[0].id}` was not working
-                const discoversSubscription = discovers.waitForFinish();
                 supabaseClient
-                    .from(TABLES.DISCOVERS)
-                    .insert([
-                        {
-                            capture_name: captureName,
-                            endpoint_config: specFormData,
-                            connector_tag_id: captureImage,
-                        },
-                    ])
+                    .from(TABLES.DRAFTS)
+                    .insert({
+                        detail: captureName,
+                    })
                     .then(
-                        (response) => {
-                            if (response.data) {
-                                setFormState({
-                                    logToken: response.data[0].logs_token,
+                        (draftsResponse) => {
```

Comment:
Eventually stuff like below will probably get pulled out into a service where we can either pass callbacks or have the service return promises if we keep seeing this pattern.